lib/types/local-date.js

```diff
@@ -110,17 +129,27 @@ LocalDate.fromDate = function (date) {
 };
 
 /**
- * Creates a new instance of LocalDate using the year, month and day provided in the form: yyyy-mm-dd.
+ * Creates a new instance of LocalDate using the year, month and day provided in the form: yyyy-mm-dd or
+ * days since epoch (i.e. -1 for Dec 31, 1969).
  * @param {String} value
  */
 LocalDate.fromString = function (value) {
-  var multiplier = 1;
-  if (value[0] === '-') {
-    value = value.substring(1);
-    multiplier = -1;
+  var dashCount = (value.match(/-/g) || []).length;
+  if(dashCount >= 2) {
+    var multiplier = 1;
+    if (value[0] === '-') {
+      value = value.substring(1);
+      multiplier = -1;
+    }
+    var parts = value.split('-');
+    return new LocalDate(multiplier * parseInt(parts[0], 10), parseInt(parts[1], 10), parseInt(parts[2], 10));
+  } else if(value.match(/^-?\d+$/)) {
+    // Parse as days since epoch.
+    //noinspection JSCheckFunctionSignatures
+    return new LocalDate(parseInt(value));
```

Comment:
The result will be the same if you remove this `else if` block.
`parseInt(value)` will yield NaN if it can not be parsed, causing the `LocalDate` constructor to throw an error (as expected).